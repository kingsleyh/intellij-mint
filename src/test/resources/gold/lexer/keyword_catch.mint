module A {

  fun a() : String {
    woop()
  } catch Error.Message => error {
    error()
  }

}