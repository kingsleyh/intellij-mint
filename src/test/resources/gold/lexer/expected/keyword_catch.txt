module ('module')
WHITE_SPACE (' ')
IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
IDENT ('a')
( ('(')
) (')')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('String')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
IDENT ('woop')
( ('(')
) (')')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE (' ')
catch ('catch')
WHITE_SPACE (' ')
IDENT ('Error')
. ('.')
IDENT ('Message')
WHITE_SPACE (' ')
=> ('=>')
WHITE_SPACE (' ')
IDENT ('error')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
IDENT ('error')
( ('(')
) (')')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')